components/menu/style/index.less

```diff
@@ -484,7 +484,7 @@
   &-item-disabled,
   &-submenu-disabled {
     color: @disabled-color !important;
-    background: none;
+    background: none !important;
```

Comment:
- There is no need to add `!important`, for that `disabled` menu item is not hoverable.

Reply:
Yes, it is necessary, because there is a need to customize the behavior of menu items on the hover, and then disabled-menu item also begins to be highlighted. he shouldn't.